java_console/inifile/src/main/java/com/opensr5/ini/IniFileModel.java

```diff
@@ -180,6 +183,9 @@ private void handleFieldDefinition(LinkedList<String> list) {
     }
 
     private void registerField(IniField field) {
+        // todo: only the first occurrence should matter, but com.rusefi.ui.TuneReadWriteTest is failing when uncommented :(
+        //if (allIniFields.containsKey(field.getName()))
```

Comment:
do we need this as dead code? let's either remove bad code or keep bad code or document what will change in the future for this code to re-appear?